easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-foss-2016b.eb

```diff
@@ -0,0 +1,80 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenCV'
+version = '3.1.0'
+
+homepage = 'http://opencv.org/'
+description = """OpenCV (Open Source Computer Vision Library) is an open source computer vision
+ and machine learning software library. OpenCV was built to provide
+ a common infrastructure for computer vision applications and to accelerate
+ the use of machine perception in the commercial products."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [
+    'https://github.com/Itseez/opencv/archive/',
+    #The Hash is version dependent! see 3rdparty/ippicv/downloader.cmake
+    'https://raw.githubusercontent.com/Itseez/opencv_3rdparty/81a676001ca8075ada498583e4166079e5744668/ippicv',
+]
+
+sources = [
+    '%(version)s.zip',
+    ('ippicv_linux_20151201.tgz', 'cp %s %(builddir)s'),
+]
+
+builddependencies = [
+    ('CMake', '3.5.2'),
+]
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('zlib', '1.2.8'),
+    ('FFmpeg', '3.1.3'),
+    ('libjpeg-turbo', '1.5.0'),
+    ('libpng', '1.6.24'),
+    ('LibTIFF', '4.0.6'),
+    ('JasPer', '1.900.1'),
+    ('Java', '1.8.0_92', '', True),
+    ('ant', '1.9.7', '-Java-%(javaver)s', True),
+    ('GLib', '2.49.5'),
+    ('GTK+', '2.24.31'),
+]
+
+#The destination directory is "linux-<MD5>" see 3rdparty/ippicv/downloader.cmake
+ppicv_dir = '3rdparty/ippicv/downloads/linux-808b791a6eac9ed78d32a7666804320e/'
```

Comment:
@hajgato typo, missing leading `i`, which leads to `NameError: name 'ippicv_dir' is not defined`